app/views/hyrax/my/collections/_modal_delete_admin_set_deny.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="modal fade" id="collection-admin-set-delete-deny-modal-<%= id %>" tabindex="-1" role="dialog" aria-labelledby="delete-admin-set-collection-label">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <form class="delete-admin-set-collection-form">
+        <div class="modal-body">
+          <p>This collection is defined as an <%= title %> and is not empty. To delete an <%= title %>, you must first remove (delete or move to another collection) all items from the <%= title %>.</p>
```

Comment:
This should be defined in locale/hyrax.en.  You can pass a value to a translation.  Define translation as `"This collection is defined as an %{type_title} and is not empty. To delete an %{type_title}, you must first remove (delete or move to another collection) all items from the %{type_title}."`
Then create in modal with `t('hyrax.dashboard.my.action.delete_admin_set_deny',collection.collection_type.title)` or something very close to this.

Reply:
Good idea.  This will fix it:          <p><%= t('hyrax.dashboard.my.action.delete_admin_set_deny', title: title) %></p>